test/LintStyle.lean

```diff
@@ -76,7 +76,7 @@ set_option linter.setOption true in
 set_option debug.moduleNameAtTimeout false
 
 -- The lint does not fire on arbitrary options.
-set_option autoImplicit false
+set_option linter.dollarSyntax true
```

Comment:
I agree that changing to a different option is a good idea; I'm not too psyched about this one.
At the end of the day, I don't feel strongly about *which* new option you choose.